examples/mock/mockoon-paymentsbbvoucher.json

```
@@ -411,6 +411,418 @@
       ],
       "enabled": true,
       "responseMode": null
+    },
+    {
+      "uuid": "236f74d1-94d9-4a39-9d0c-17fb74e68302",
+      "documentation": "Request for voucher redemption by sending the voucher number, the merchant name and merchant payment details.",
+      "method": "post",
+      "endpoint": "vouchers/voucher_redeemption",
```

Just a quick check, isn't there a typo in endpoint name? I think it should be
```suggestion
      "endpoint": "vouchers/voucher_redemption",
```

If it's also the case in OpenAPI schema please check if this was intentional.